### PR TITLE
fix(security): fix explaination about enforce password policy

### PR DESCRIPTION
**Pull request by tbouffard**
Use the dependency that exists in 7.3+ version and explain how to
retrieve the dependency manually.

Closes [BS-17598](https://bonitasoft.atlassian.net/browse/BS-17598)

### CODE REVIEW

**Comment by tbouffard** [comment]
I have tested the procedure by testing it using the attached test project [bonita-web-alternate-password-policy.zip](https://github.com/bonitasoft/bonita-doc/files/1505808/bonita-web-alternate-password-policy.zip)
It works for 7.3.x, 7.4.x, 7.5.x, 7.6.x and 7.7.x

File: md/enforce-password-policy.md [hunk 1]
```diff
@@ -41,13 +41,28 @@ Here are the steps to add a custom password validator:
...
+**Note:** As this artifact is not available in public repositories, you
+have to follow this given procedure (example given for the 7.3.3 version):
+* retrieve the console-common-7.3.3.jar from a Bonita distribution or live
+instance (in the bonita.war or in the explosed war)
+* retrieve poms from the github repository
+  * bonita-console: https://github.com/bonitasoft/bonita-web/blob/7.3.3/pom.xml
+  * console-common: https://github.com/bonitasoft/bonita-web/blob/7.3.3/common/pom.xml
+* manually put the jars and pom in your local repository or deploy them
+into your repository manager
+  * bonita-common pom.xml file in <M2_REPO>/org/bonitasoft/console/console-common/7.3.3
```
Comment (ugaston) [inline md/enforce-password-policy.md]: I guess you meant: console-common (instead of bonita-common).

**Review by ugaston — COMMENTED** [review]
But for one tiny comment, it looks fine to me.

Cheers!

File: md/enforce-password-policy.md (hunk 1 above)
Comment (tbouffard) [reply md/enforce-password-policy.md]: yes, fixed

**Review by ugaston — APPROVED** [review]
Fine by me.

Thanks.

**Comment by tnguyen1** [comment]
retest this please

**Review by abirembaut — COMMENTED** [review]
LGTM

**Pull request merged**